BitwardenShared/Core/Platform/Services/ConfigService.swift

```diff
@@ -6,16 +7,21 @@ import OSLog
 /// A protocol for a `ConfigService` that manages the app's config.
 ///
 protocol ConfigService {
+    /// A publisher that publishes a new value when a new server config has been gotten.
```

Comment:
🎨 "A publisher that updates with a new value when a new server configuration is received."
What do you think?